TASK-2_1/Chatbot.cs

```diff
@@ -0,0 +1,61 @@
+using System.Text.RegularExpressions;
+
+namespace Task2_1;
+
+public class Chatbot
+{
+    public string UserName { get; set; }
+    public string UserSurname { get; set; }
+
+    public DateTime Date { get; set; }
+
+    public static string InputName(int nameType)
```

Comment:
Use strings "name" and "username" instead of int for your logic. It's more human readable

Reply:
Ok